bicycleparameters/main.py

```diff
@@ -726,6 +728,271 @@ def com_symbol(ax, center, radius, color='b'):
 
         return fig
 
+
+    def plot_bicycle_geometry_plotly(self, show=True, pendulum=True,
+                              centerOfMass=True, inertiaEllipse=True):
+        """Returns a figure showing the basic bicycle geometry, the centers of
+        mass and the moments of inertia.
+
+        Parameters
+        ==========
+        show : boolean, optional
+            If true ``matplotlib.pyplot.show()`` will be called before exiting
+            the function.
+        pendulum : boolean, optional
+            If true the axes of the torsional pendulum will be displayed (only
+            useful if raw measurement data is availabe).
+        centerOfMass : boolean, optional
+            If true the mass center of each rigid body will be displayed.
+        inertiaEllipse : boolean optional
+            If true inertia ellipses for each rigid body will be displayed.
+
+        Returns
+        =======
+        fig : matplotlib.pyplot.Figure
```

Comment:
This returns some kind of plotly object. You can figure that out with `fig = bike.plot_geom_plotly(); type(fig)`.